package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "npm run lint && npm run unit && npm run check-coverage && npm audit --production",
     "lint": "eslint .",
-    "unit": "nyc --reporter=lcov --reporter=text-summary _mocha test/spec/ --recursive --require test/helper",
+    "unit": "nyc --reporter=lcov --reporter=text-summary _mocha test/spec/ --recursive --require test/helper.js",
     "check-coverage": "nyc check-coverage"
   },
```

Comment:
Just noticed we're missing the husky `prepare` script here as husky is completely new to this lib. Should be able to get husky to add it itself by running `npx husky init`. That might also add or modify hooks in `.husky/` though, if it does we can just remove those changes because what we've got for the hooks now is what we want.